### PR TITLE
push official images to quay.io in addition to docker.io

### LINKED ISSUE #2498: Push official images to quay.io, in addition to Docker Hub
# Requirement - what kind of business use case are you trying to solve?

High availability and resiliency of Jaeger in production environments

## Problem - what in Jaeger blocks you from solving the requirement?

Currently, the official images for jaeger are on docker hub. Unless one has a docker paid account, image pulls will be rate-limited and pulls may fail at run-time, causing infrastructure issues.

## Proposal - what do you suggest to solve the problem or improve the existing situation?

$title: Push official images to quay.io, in addition to Docker Hub

We already have an organization where we push the jaeger operator https://quay.io/repository/jaegertracing/jaeger-operator

I propose that we also push all the other images to quay.io, in addition to the docker hub repository here: https://hub.docker.com/u/jaegertracing

## Any open questions to address

None

### PR DESCRIPTION
**Pull request by aebirim**
Resolves #2498

## Which problem is this PR solving?
-  pushes official images to quay.io in addition to docker.io

## Short description of the changes
- addition of upload-to-quay script which pushes docker images to the quay.io platform

### CODE REVIEW

**Comment by albertteoh** [comment]
Please remove `.DS_Store` files from the PR. 

I also recommend in future, to make changes in a branch rather than `master` of your fork; keep your `master` "pristine" and updated with the upstream jaegertracing/jaeger master.

File: scripts/travis/upload-to-quay.sh [hunk 1]
```diff
@@ -0,0 +1,70 @@
+#!/bin/bash
```
Comment (albertteoh) [inline scripts/travis/upload-to-quay.sh:1]: Did you see @jpkrohling's comment? https://github.com/jaegertracing/jaeger/pull/2325/files#r509103626

I think you may not need this script; and should just need to modify the aforementioned script once #2325 is merged.

File: .gitignore [hunk 2]
```diff
@@ -37,3 +37,4 @@ cmd/opentelemetry/cmd/agent/opentelemetry-agent-*
 cmd/opentelemetry/cmd/ingester/opentelemetry-ingester-*
 cmd/opentelemetry/cmd/all-in-one/opentelemetry-all-in-one-*
 __pycache__
+.DS_Store
```
Comment (albertteoh) [inline .gitignore:40]: I would prefer this be added to your local `~/.gitexcludes` file as there should not be a reason to commit this file into a repository, and it's MacOS specific.

**Comment by aebirim** [comment]
> Please remove `.DS_Store` files from the PR.
> 
> I also recommend in future, to make changes in a branch rather than `master` of your fork; keep your `master` "pristine" and updated with the upstream jaegertracing/jaeger master.

@albertteoh .DS_Store removed as requested

File: scripts/travis/upload-to-quay.sh (hunk 1 above)
Comment (aebirim) [reply scripts/travis/upload-to-quay.sh:1]: no worries. I'll add my changes as soon as #2325 is merged

File: .gitignore (hunk 2 above)
Comment (aebirim) [reply .gitignore:40]: > I would prefer this be added to your local `~/.gitexcludes` file as there should not be a reason to commit this file into a repository, and it's MacOS specific.

added DS_Store to my .gitexcludes file

**Comment by yurishkuro** [comment]
Done in #2783

**Pull request closed without merging**